tests/rbd_mirror/test_9501_del_primary_im.py

```diff
@@ -0,0 +1,74 @@
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
```

Comment:
IMO, we could rename to module to `test-image-delete-primary-site.py`

Reply:
Sorry for the name format error
Changed it, Thank you